db/seeds.rb

```diff
@@ -18,6 +18,7 @@
     beginning_at: date,
     end_at: date + 2.days,
     category: categories.sample,
-    description: Faker::HipsterIpsum.words(250).join(' ').capitalize
+    description: Faker::HipsterIpsum.words(250).join(' ').capitalize,
+    user_id: rand(1..5)
```

Comment:
Yes, but we don't need to have any right now. We log in with FB which creates user in our Database. We just want to have our own events. Seed events are connected to users with id form 1 to 5. After logging to meecon app via FB we'll be first users and it's highly possible that our id will be betwen 1 and 5.